### ISSUE TITLE
Should RxJava example written in Java?

### ISSUE THREAD

**Issue by tonywei92** [ReactiveX/RxJava#5835]
Hello developer of RxJava, i'm grateful you made this RxJava, it's very impressive, but i wanna ask about this article, https://github.com/ReactiveX/RxJava/wiki/How-To-Use-RxJava

The example is good, but i'm just curious, why example not written in Java while the name is RxJava? because it's bit harder for me to test the code, i saw "def" and "defn" keyword which is not belong to Java, could you please elaborate? thank you.

**Comment by akarnokd** [comment]
That page was contributed a long time ago when the RxJava repository hosted language adapters and some contributors were mainly interested in those other JVM languages. As the adapters were moved into their separate repositories, the wiki was simply not updated (and most contributors simply left).

I promised once development on RxJava 1 stops this spring, I (we) will start a review and rework of the wiki pages.
